tycho-core/src/test/java/org/eclipse/tycho/core/test/TychoTest.java

```diff
@@ -113,47 +110,48 @@ public void testProjectPriority() throws Exception {
         assertEquals("0.0.1", dependency.getVersion());
     }
 
-    public void testFragment() throws Exception {
-
-        File basedir = getBasedir("projects/fragment");
-
-        List<MavenProject> projects = getSortedProjects(basedir);
-
-        List<String> artifactIds = new ArrayList<>();
-        for (MavenProject project : projects) {
-            artifactIds.add(project.getArtifactId());
-        }
-        assertEquals(0, artifactIds.indexOf("parent"));
-        assertEquals(artifactIds.size() - 1, artifactIds.indexOf("client"));
-        assertTrue(artifactIds.indexOf("host") < artifactIds.indexOf("fragment2"));
-        assertTrue(artifactIds.indexOf("host") < artifactIds.indexOf("fragment")
-                && artifactIds.indexOf("dep") < artifactIds.indexOf("fragment"));
-
-        // host does not know anything about fragments
-        MavenProject host = projects.stream().filter(p -> p.getArtifactId().equals("host")).findAny().get();
-        List<Dependency> hostDependencies = host.getModel().getDependencies();
-        assertEquals(0, hostDependencies.size());
-
-        MavenProject fragment = projects.stream().filter(p -> p.getArtifactId().equals("fragment")).findAny().get();
-        List<Dependency> fragmentDependencies = fragment.getModel().getDependencies();
-        // host first, then fragment dependency
-        assertEquals(2, fragmentDependencies.size());
-        assertEquals("host", fragmentDependencies.get(0).getArtifactId());
-        assertEquals("dep", fragmentDependencies.get(1).getArtifactId());
-
-        // host only
-        MavenProject fragment2 = projects.stream().filter(p -> p.getArtifactId().equals("fragment2")).findAny().get();
-        List<Dependency> fragment2Dependencies = fragment2.getModel().getDependencies();
-        assertEquals(1, fragment2Dependencies.size());
-        assertEquals("host", fragment2Dependencies.get(0).getArtifactId());
-
-        // depends on host and because host has ExtensibleAPI also depends fragment and fragent2
-        MavenProject client = projects.stream().filter(p -> p.getArtifactId().equals("client")).findAny().get();
-        List<Dependency> clientDependencies = client.getModel().getDependencies();
-        assertEquals(4, clientDependencies.size());
-        assertEquals(Set.of("host", "fragment", "dep", "fragment2"),
-                clientDependencies.stream().map(Dependency::getArtifactId).collect(Collectors.toSet()));
-    }
+//    @Ignore
+//    public void testFragment() throws Exception {
```

Comment:
@mickaelistria I think this is a left-over from the old PDE based resolver and relies on the ordering of fragments by the ExtensibleAPI header, I think we can remove this:

1. We no longer use the old PDE used resolver but "upgraded" to new equinox one
2. the test uses the P2StubResolver what still uses the old code, so this does not really test what Tycho do in a build
3. ExtensibleAPI is nowhere else referenced, and even disabled in the p2StubResolver